samples/bluetooth/peripheral_uart/prj.conf

```diff
@@ -13,8 +13,7 @@ CONFIG_CONSOLE=y
 CONFIG_CONSOLE_HAS_DRIVER=y
 CONFIG_UART_CONSOLE=y
 
-# Smalles possible selectable heap memory pool size
-CONFIG_HEAP_MEM_POOL_SIZE=256
+CONFIG_HEAP_MEM_POOL_SIZE=2048
```

Comment:
Increased Heap size to match central uart - more heap size is required in the OpenAMP library.